### DISCUSSION TITLE
Companion application required  This operation requires an external application to be completed. Click here to troubleshoot your issue.

### DISCUSSION THREAD

**Discussion by irodionzaytsev** [aclap-dev/video-downloadhelper#1270] [Help and Q&A]
### Your question

How can I fix the following issue:
Companion application required

This operation requires an external application to be completed. [Click here to troubleshoot your issue.](https://github.com/aclap-dev/video-downloadhelper/wiki/CoApp-not-recognized)

I just downloaded firefox via snap and vdh with the curl command, so they are latest version.

### Addon & CoApp version and details

Version 9.1.0.44 (stable), target mozilla, locale en-US.

**Comment by Mattmjs** [comment]
Did you install the companion app ?
